src/lib/Grid.svelte

```diff
@@ -1,93 +1,84 @@
+    muuriConfig?: GridOptions;
+    class?: string;
+    children: Snippet;
+  }
+  export function refreshItems() {
+    grid?.refreshItems();
+  }
 
-	let { muuriConfig, class: className = '', children }: Props = $props();
+  let { muuriConfig, class: className = "", children }: Props = $props();
 
-	let gridElement: HTMLDivElement;
-	let grid: any = null;
+  let gridElement: HTMLDivElement;
+  let grid: any = null;
 
-	onMount(async () => {
-		const MuuriConstructor = (await import('muuri')).default;
+  onMount(async () => {
+    const MuuriConstructor = (await import("muuri")).default;
 
-		const config: GridOptions = muuriConfig || {
-			dragEnabled: true,
-			layout: {
-				fillGaps: true,
-				rounding: true,
-				horizontal: false
-			},
-			layoutOnInit: true,
-			dragStartPredicate: function (item: Item, e: any) {
-				const target = e.srcEvent.target as HTMLElement;
-				const isResizeHandle = target.closest('[data-resize-handle]') !== null;
+    const config: GridOptions = muuriConfig || {
+      dragEnabled: true,
+      layout: {
+        fillGaps: true,
+        rounding: true,
+        horizontal: false
+      },
+      layoutOnInit: true,
+      dragStartPredicate: function (item: Item, e: any) {
+        const target = e.srcEvent.target as HTMLElement;
+        const isResizeHandle = target.closest("[data-resize-handle]") !== null;
 
-				if (isResizeHandle) {
-					return false;
-				}
+        if (isResizeHandle) {
+          return false;
+        }
 
-				return (MuuriConstructor as any).ItemDrag.defaultStartPredicate(item, e);
-			},
-			dragPlaceholder: {
-				enabled: true,
-				createElement: (item: Item) => {
-					const node = item.getElement()?.cloneNode(true) as HTMLElement;
-					node.style.opacity = '0.5';
-					node.style.margin = '0';
-					return node;
-				}
-			}
-		};
+        return (MuuriConstructor as any).ItemDrag.defaultStartPredicate(item, e);
+      },
+      dragPlaceholder: {
+        enabled: true,
+        createElement: (item: Item) => {
+          const node = item.getElement()?.cloneNode(true) as HTMLElement;
+          node.style.opacity = "0.5";
+          node.style.margin = "0";
+          return node;
+        }
+      }
+    };
 
-		grid = new (MuuriConstructor as any)(gridElement, config);
-		console.log(grid);
-	});
+    grid = new (MuuriConstructor as any)(gridElement, config);
+    console.log(grid);
+  });
 
-	// Handle cleanup separately
-	onMount(() => {
-		return () => {
-			grid?.destroy();
-		};
-	});
+  // Handle cleanup separately
+  onMount(() => {
+    return () => {
+      grid?.destroy();
+    };
+  });
 
-	export function refresh() {
-		if (!grid) return;
-		grid.refreshItems();
-		grid.layout();
-	}
+  export function refresh() {
+    if (!grid) return;
+    grid.refreshItems();
+    grid.layout();
+  }
 
-	setContext('grid', {
-		refresh: refresh
-	});
+  setContext("grid", {
+    refresh: refresh
+  });
 </script>
 
 <div
-	bind:this={gridElement}
-	class={twMerge(
```

Comment:
ahh my bad.. I didn't mean ALL css.. I just was referring to the non "required" stuff that sets things up at a minimum like background color and shadow.